doc/sphinx/proof-engine/ltac.rst

```diff
@@ -1193,25 +1177,38 @@ Pattern matching on terms: match
    .. todo There's a more realistic example from @JasonGross here:
       https://github.com/coq/coq/pull/12103#discussion_r432996954
 
-   .. todo @term after context is actually @lconstr_pattern (term with holes), which is spliced away
-      Maybe worth restoring that distinction?
+   :n:`@ltac_expr`
+      The tactic to apply if the construct matches.  Metavariable values from the pattern
+      match are substituted
+      into :n:`@ltac_expr` before it's applied.  Note that metavariables are not
+      prefixed with the question mark as they are in :token:`cpattern`.
+
+      If :token:`ltac_expr` evaluates to a tactic, then it is
+      applied. If the tactic succeeds, the result of the match expression is
+      :tacn:`idtac`.  If :token:`ltac_expr` does not evaluate
+      to a tactic, that value is the result of the match expression.
+
+      If :n:`@ltac_expr` is a tactic with backtracking points, then subsequent
+      failures (after the `*match*` construct) can cause this tactic to backtrack.
```

Comment:
This is false.
```suggestion
      failures (after the `*match*` construct) will only cause this tactic to backtrack if the `match` construct is a `multimatch` or `lazymatch`.
```
```coq
Goal True.
  try (match goal with _ => idtac "A" + idtac "B" end; fail). (* A *)
  try (lazymatch goal with _ => idtac "A" + idtac "B" end; fail). (* A B *)
  try (multimatch goal with _ => idtac "A" + idtac "B" end; fail). (* A B *)
```

Reply:
That's because `match` is implemented as `once multimatch`, right?

I have this now:

      If :n:`@ltac_expr` is a tactic with backtracking points, then subsequent
      failures after a :tacn:`lazymatch` or :tacn:`multimatch` (but not :tacn:`match`) can cause
      backtracking into :n:`@ltac_expr` to select its next success.
      (:tacn:`match` :n:`…` is equivalent to :tacn:`once` :tacn:`multimatch` `…`.
      The :tacn:`once` prevents backtracking into the :tacn:`match` after it has succeeded.)

Reply:
Sounds good! 👍

Reply:
And yes, that's correct.